docs/can-canjs/can-infrastructure.md

```diff
@@ -298,7 +298,7 @@ var unicorn = {
 if (namespace.unicorn) {
 	throw new Error("You can't have two versions of can-unicorn, check your dependencies");
 } else {
-	module.exports = namespace.unicorn = unicorn;
+	export default namespace.unicorn = unicorn;
```

Comment:
You can't have exports inside of if statements with es6. We'll have to figure out another way to do this.